ui/src/types/updates.types.ts

```diff
@@ -53,3 +55,16 @@ export enum UpdateStatus {
   UpToDate = 'up-to-date',
   UpdateClients = 'update-clients',
 }
+
```

Comment:
We will have conflicts with https://github.com/percona/grafana/pull/776/files#diff-336058a35246048f36019d0f034d8e56bfec5bfa5f32be6db5797821c456463a

I've also used the camel case for Changelog everywhere it is used